less/project/columns.less

```diff
@@ -0,0 +1,32 @@
+.make-columns(@size, @media-query, @count, @iteration: 1) when (@count >= @iteration) {
```

Comment:
In my initial prototype for this I included a 0 column width for instances where we might want to include a blank component to create white space for specific view widths. I can't see the need for a 0 width other than with a blank component so we could target 'blank' exclusively for this. I'll have a look into this and make a suggestion...

Reply:
There's some usefulness to it but it's definitely a small window with the current device width break points. When we increase these break points the window for this use case will increase.

By default all components are 100% width. With the xs 0 column width selector you could change the size of a component between 0 and the sm device width small selector (or larger) to be a value that's not 100%.